config/params/mainnet_config.go

```diff
@@ -237,6 +237,6 @@ var mainnetBeaconConfig = &BeaconChainConfig{
 	// Light client
 	MinSyncCommitteeParticipants: 1,
 
-	// Merge
+	// Bellatrix
```

Comment:
This is a close one, but I think I like `Merge` better here. `TerminalBlockHashActivationEpoch` is used for merging the two chains

Reply:
Done

Reply:
@leolara it still says Bellatrix. Did you intend to change it back to "Merge" per Terence's comment?

Reply:
Done